Rakefile

```diff
@@ -35,24 +33,25 @@ namespace :test do
     t.libs << 'test'
     t.verbose = true
   end
-
 end
 
-desc "Delete tar.gz / zip"
-task :cleanup => [ :clobber_package ]
+desc 'Delete tar.gz / zip'
+task cleanup: %i[clobber_package]
 
-spec = eval(File.read('activemerchant-realex3ds.gemspec'))
+# TODO: refactor to allow for build task but without using eval
+#
+# spec = eval(File.read('activemerchant-realex3ds.gemspec'))
```

Comment:
can we remove these or these kept for reference?

Reply:
Turns out if you call `Gem::Specification.load(file)` or do something like require `bundler/gem_tasks`, then they will both still eval the file, so just choosing to keep this as is, but call `Gem::Specification.load(file)` to lean on a loading mechanism that is well maintained.